samples/libzstd/compile.sh

```diff
@@ -0,0 +1,6 @@
+jextract -t libzstd \
+  -I /usr/local/Cellar/zstd/1.5.2/include \
```

Comment:
Consider replacing the path `/usr/local/Cellar/zstd/1.5.2` with `$(brew --prefix zstd)` or `$(pkg-config --variable=prefix libzstd)`